.buildkite/pipeline.yml

```diff
@@ -1,7 +1,7 @@
 steps:
   - commands:
       - ". $HOME/.asdf/asdf.sh"
-      - "asdf local erlang 21.3"
+      - "asdf local erlang 22.1.8"
```

Comment:
do we need to do this?  is this what we're currently shipping with?

Reply:
yeah professor added a call to a v22 API last week ( gen_udp:send/3) on the lora PR.  I hit this today as part of this PR with dialyzer failing and so this fix is to rectify that ( whilst the core prob is not related to this PR )

Reply:
We are shipping OTP 22 on the hotspots